pkg/minikube/machine/advice.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package machine
+
+import (
+	"runtime"
+
+	"github.com/pkg/errors"
+	"k8s.io/minikube/pkg/drivers/kic/oci"
+	"k8s.io/minikube/pkg/minikube/out"
+)
+
+// MaybeAdviceNoExit will provide advice without exiting, so minikube has a chance to try the failover
+func MaybeAdviceNoExit(err error, driver string) {
+	if errors.Is(err, oci.ErrDaemonInfo) {
+		out.ErrLn("")
+		out.ErrT(out.Conflict, "{{.driver_name}} couldn't proceed because {{.driver_name}} service is not healthy.", out.V{"driver_name": driver})
+	}
+
+	if errors.Is(err, oci.ErrExitedAfterCreate) {
+		out.ErrLn("")
+		out.ErrT(out.Conflict, "Unfortunately {{.driver_name}} container exited after it was created, with an unclear root cause.", out.V{"driver_name": driver})
```

Comment:
Remove the `Unfortunately` - it's extra wording that does not help in context. See: 

https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language

My suggestion is something shorter, like:

`The minikube 'docker' container exited unexpectedly`.

Reply:
done